src/Findologic/Request/FindologicRequestFactory.php

```diff
@@ -127,6 +110,31 @@ private function fetchClientIp(): string
         return $ipAddress;
     }
 
+    /**
+     * @throws InvalidArgumentException
+     * @throws InconsistentCriteriaIdsException
+     */
+    protected function getPluginVersion(): string
```

Comment:
Why was this method moved here? If your intention was to move the `setDefaults` to the top, then please move this method above `fetchClientIp`, as it is a private method.

Reply:
I had no intentions to move it anywhere, now it is above the `fetchClientIp` method

Reply:
This wasn't resolved.

Reply:
@TheKeymaster Do I have to make them in sequence? like, all protected functions then all private functions?

Reply:
@D3xter7 either that or you revert the change, if you didn't intend to change it after all.

Reply:
@TheKeymaster I have made their sequence, protected -> private.